ts/torch_handler/base_handler.py

```diff
@@ -149,6 +149,8 @@ def initialize(self, context):
             self.device = torch.device(
                 self.map_location + ":" + str(properties.get("gpu_id"))
             )
+        elif hasattr(self, "model_yaml_config") and "mps" in self.model_yaml_config and self.model_yaml_config["mps"] == "enable":
```

Comment:
Can we add `mps` here https://github.com/pytorch/serve/blob/1a99de4675eaaad4ddf532e603ae2f175656462a/model-archiver/README.md?plain=1#L174

cc @lxning

Reply:
In terms of design, having one config for all `deviceType` probably makes sense?